monix-execution/shared/src/test/scala/monix/execution/misc/LocalSuite.scala

```diff
@@ -73,8 +75,10 @@ object LocalSuite extends SimpleTestSuite {
     local2 := 100
 
     val f = Local.isolate {
-      local1 := 100
-      Future(local1.get + local2.get)
+      Future {
```

Comment:
with current changes
```scala
Local.isolate {
  local1 := 100
  Future (...)
}
```
currently goes for `Future` variant but doesn't isolate properly :( unless we stick modification inside
```

Reply:
fixed both cases